homeassistant/components/esphome/light.py

```diff
@@ -388,3 +411,10 @@ def _on_static_info_update(self, static_info: EntityInfo) -> None:
         self._attr_effect_list = static_info.effects
         self._attr_min_mireds = round(static_info.min_mireds)
         self._attr_max_mireds = round(static_info.max_mireds)
+        self._attr_min_color_temp_kelvin = _mired_to_kelvin(static_info.max_mireds)
+        self._attr_max_color_temp_kelvin = _mired_to_kelvin(static_info.min_mireds)
+
+    @property
+    def effect_list(self) -> list[str]:
+        """Return the list of supported effects."""
+        return self._static_info.effects
```

Comment:
```suggestion
```

Reply:
I see what you mean now. These lines should be safe to remove.